Languages/IronPython/Tests/test_privateBinding.py

```diff
@@ -129,6 +129,6 @@ def test_misc_coverage():
 
 run_test(__name__, noOutputPlease=True)
 
-if not privateBinding and not is_silverlight:
+if not privateBinding and not is_silverlight and not is_netstandard: # sys.executable isn't an executable
```

Comment:
I'm not sure, perhaps I could make it point to a script that could serve as a launcher (e.g. batch file on Windows).